keyboards/ghost_spot/ghost54/keymaps/default/keymap.c

```diff
@@ -0,0 +1,157 @@
+ * | ESC  |      |  F10 |  F11 |  F12 |      |                    |      |  F22 |  F23 |  F24 |      | Pscr |
+ * |------+------+------+------+------+------|                    |------+------+------+------+------+------|
+ * | Tab  |      |  F7  |  F8  |  F9  |      |                    |      |  F19 |  F20 |  F21 |      |      |
+ * |------+------+------+------+------+------|                    |------+------+------+------+------+------|
+ * |LShift|      |  F4  |  F5  |  F6  |      |-------.    ,-------|      |  F16 |  F17 |  F18 |      |      |
+ * |------+------+------+------+------+------|WIN+TAB|    |  Play |------+------+------+------+------+------|
+ * | Ctrl |      |  F1  |  F2  |  F3  |      |-------|    |-------|      |  F13 |  F14 |  F15 |      |      |
+ * `-----------------------------------------/       /     \      \-----------------------------------------'
+ *                          | Enter|Space | /TO(1)  /       \ TO(0)\  |Bcksp | Del  |
+ *                          |      |      |/       /         \      \ |      |      |
+ *                          `---------------------'           '------''-------------'
+ */
+[3] = LAYOUT(
+  KC_ESC,  _______,   KC_F10, KC_F11, KC_F12,_______,              _______, KC_F22, KC_F23, KC_F24, _______, KC_PSCR,
+  KC_TAB,  _______,   KC_F7,  KC_F8,  KC_F9, _______,              _______, KC_F19, KC_F20, KC_F21, _______, _______,
+  KC_LSFT, LGUI_T(_______),   LALT_T(KC_F4),  LCTL_T(KC_F5),  LSFT_T(KC_F6), _______,              _______, RSFT_T(KC_F16), LCTL_T(KC_F17), LALT_T(KC_F18), LGUI_T(_______), _______,
+  KC_LCTL, _______,   KC_F1,  KC_F2,  KC_F3, _______,              _______, KC_F13, KC_F14, KC_F15, _______, _______,
+                      KC_ENT,  KC_SPC, TO(1),  LGUI(KC_TAB),        KC_MPLY, TO(0),  KC_BSPC,KC_DEL
+)
+};
+
+
+// Combos
+
+const uint16_t PROGMEM layer0to3[] = {TO(1), MO(2), COMBO_END};
+const uint16_t PROGMEM layer1to3[] = {TO(0), MO(2), COMBO_END};
+const uint16_t PROGMEM layer2to3[] = {TO(1), TO(0), COMBO_END};
+combo_t key_combos[] = {
+    COMBO(layer0to3, TO(3)),
+    COMBO(layer1to3, TO(3)),
+    COMBO(layer2to3, TO(3)),
+};
+
+
+#ifdef OLED_ENABLE
+
+// Draw to OLED
+bool oled_task_user(void) {
+
+    // Switch on current active layer
+    switch (get_highest_layer(layer_state)) {
+        case _WIN_DEFAULT :
+            oled_write("Main Layer", false);
+            break;
+        case _WIN_NUMPAD :
+            oled_write("Numpad Layer", false);
+            break;
+        case _WIN_SYMBOLS :
+            oled_write("Symbols Layer", false);
+            break;
+        case _WIN_FKEYS :
+            oled_write("Function Layer", false);
+            break;
+    }
+
+    // Caps Lock Status
+    led_t led_state = host_keyboard_led_state();
+    oled_write_P(led_state.caps_lock ? PSTR("Caps Lock On") : PSTR("Caps Lock Off"), false);
+
+    return false;
+}
```

Comment:
This oled code should be moved to the keyboard level (ghost54.c in the ghost54 folder), and the function should be renamed/updated to start with: 
```c
bool oled_task_kb(void) {
    if (!oled_task_user()) {
        return false;
    }
    ...
```